src/libraries/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -1156,6 +1156,7 @@ private void SetSessionHandleConnectionOptions(SafeWinHttpHandle sessionHandle)
 
         private void SetSessionHandleTlsOptions(SafeWinHttpHandle sessionHandle)
         {
+            const SslProtocols Tls13 = (SslProtocols)12288; // enum is missing in .NET Standard
```

Comment:
```suggestion
            const SslProtocols Tls13 = (SslProtocols)0x3000; // enum is missing in .NET Standard
```

Reply:
I took the value from [documentation](https://docs.microsoft.com/en-us/dotnet/api/system.net.securityprotocoltype?view=net-5.0#System_Net_SecurityProtocolType_Tls13). But I don't really care. It seems like the SP_PROT is platform specific mapping that happens to be the same on Windows. 
It feels like this is really matter of personal preference.